bin/bootstrap-env.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 
 WP_VERSION=${WP_VERSION-latest}
+WP_DEBUG=${WP_DEBUG-true}
```

Comment:
Possible thought on the error: Is this the correct syntax for assigning the default? Is the colon `:` not required?

See:

- https://stackoverflow.com/a/2013589
- https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Reply:
Also a bit curious why we don't use a `.env` file: https://docs.docker.com/compose/env-file/

Reply:
I followed other occurrences in the file. I think it works correctly, although I personally don't know which syntax would be preferred.

The issue was caused by the fact that PHP didn't have those constants defined and WP CLI's implementation is limited to updating existing constants ... https://github.com/WordPress/gutenberg/pull/14371/commits/c34d3c027aea9ddd06639bb5a4b7e9b38cc5ce7f fixes it.

Reply:
Yes, this is complex to explain but correct. Each constant needs to be defined in `wp-config.php` if we want to be able to update it with WP-CLI.